builds/fabric8-che/assembly/assembly-wsagent-war/src/main/java/org/eclipse/che/wsagent/server/WsAgentAnalyticsAddresser.java

```diff
@@ -1,46 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2012-2017 Codenvy, S.A.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- *
- * Contributors:
- *   Codenvy, S.A. - initial API and implementation
- *******************************************************************************/
-package org.eclipse.che.wsagent.server;
```

Comment:
As there is no change at the moment, we don't add it. If we need in the future something like this, let's add it at this time.
And replacing the class is not a good way to do anyway.
At the moment this is adding one more class to maintain for nothing.